sdk/core/core-rest-pipeline/package.json

```diff
@@ -82,7 +82,16 @@
         "path": "src/constants.ts",
         "prefix": "SDK_VERSION"
       }
-    ]
+    ],
+    "sampleConfiguration": {
```

Comment:
```suggestion
    "//metadata": {
```

Reply:
//metadata already exists and is parent of this

Reply:
Oh I see, marking this as resolved.